30_ComputeShaderPathTracer/app_resources/present.frag

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2024-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#pragma wave shader_stage(fragment)
+
```

Comment:
the reason this shader won't load as a bunch of HLSL is because of missing `.hlsl` extension, use `.frag.hlsl`

P.S. The `.frag` is redundant if you already have `#pragma wave shader_stage`

Reply:
Done